distinct-elements.js

```diff
@@ -0,0 +1,15 @@
+var array = [0, 5, 10, 3, 5, 12, 5, 3, 9, 1, 0, 5];
+for (i = 0; i < array.length; i++) {
+    count = 0;
+    for (j = 0; j < array.length; j++) {
+        if(j<i && array[i] == array[j]){
+            break;
+        }
+        else if (array[i] == array[j]) {
+             count++;
+        }
+    }
+    if(count!=0){
+        console.log(array[i] + "-" + count);
+    }
+}
```

Comment:
another solution to this problem is to sort the array. the it becomes easier to count.

Reply:
does sorting the array reduces the execution time in counting the duplicate ?

Reply:
It depends. Sorting algorithms are very fast now a days. no one uses bubble sort now a days.  If your case, what if the array is already sorted ? it will still go ahead and do the looping.